ppr-ui/src/composables/mhrRegistration/home-owner.ts

```diff
@@ -284,22 +291,32 @@ export function useHomeOwners (isPerson: boolean = false, isEditMode: boolean =
     return { updatedGroups, updatedFractionalData }
   }
 
+  const showGroupHeading = (): Boolean => {
+    return getMhrRegistrationHomeOwnerGroups.value.length === 0 ||
+    ((getMhrRegistrationHomeOwnerGroups.value.length === 1 && !getMhrRegistrationHomeOwners.value.address) &&
+      getMhrRegistrationHomeOwnerGroups.value[0].owners.length <= 1 && getMhrRegistrationHomeOwnerGroups.value[0].interestNumerator === null)
+  }
+
```

Comment:
A small comment describing this chain of operators would be useful to other devs i think

Reply:
Good suggestion..